test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsSqsTests.cs

```diff
@@ -0,0 +1,169 @@
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Core.Tools;
+using Datadog.Trace.ClrProfiler.IntegrationTests.Helpers;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AWS
+{
+    public class AwsSqsTests : TestHelper
+    {
+        private static List<MockTracerAgent.Span> _expectedSpans = new()
+        {
+#if NETFRAMEWORK
+            // CreateSqsQueue
+            AwsSqsSpan.GetDefault("CreateQueue")
+                    .WithTag("aws.queue.name", "MySQSQueue"),
+            AwsSqsSpan.GetDefault("CreateQueue")
+                    .WithTag("aws.queue.name", "MySQSQueue2"),
+
+            // SendMessagesWithInjectedHeaders
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // SendMessagesWithoutInjectedHeaders
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // SendMessage
+            AwsSqsSpan.GetDefault("SendMessage"),
+            AwsSqsSpan.GetDefault("SendMessage"),
+
+            // ReceiveMessageAndDeleteMessage
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessage"),
+
+            // SendMessageBatch
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+            AwsSqsSpan.GetDefault("SendMessageBatch"),
+
+            // ReceiveMessagesAndDeleteMessageBatch
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+            AwsSqsSpan.GetDefault("ReceiveMessage"),
+            AwsSqsSpan.GetDefault("DeleteMessageBatch"),
+
+            // DeleteQueue
+            AwsSqsSpan.GetDefault("DeleteQueue"),
+            AwsSqsSpan.GetDefault("DeleteQueue"),
+
+#endif
+            // CreateSqsQueueAsync
+            AwsSqsSpan.GetDefault("CreateQueue")
+                    .WithTag("aws.queue.name", "MySQSQueue"),
+            AwsSqsSpan.GetDefault("CreateQueue")
+                    .WithTag("aws.queue.name", "MySQSQueue2"),
```

Comment:
I'm guessing these should all have the `Async` suffix 🙂

Reply:
Good eye, but this proposal was intentional to not have the `Async` suffix. While the .NET SDK has the `Async` suffix on method names, the SQS API does not have that suffix so I wanted to pair 1:1 with the actual API operations. Do you think that's the correct approach or we should match this to the client SDK method name?

Reply:
> Good Eye

Except I missed that the `OperationName` doesn't have the `*Async` suffix in the instrumentation 😆 

> Do you think that's the correct approach or we should match this to the client SDK method name?

That's a good question. I think your intuition is correct - given this is about resource names, the "resource" is identical, regardless of whether you're using the `CreateQueue` or `CreateQueueAsync` method.